### PR TITLE
[SPARK-24810][SQL] Fix paths to test files in AvroSuite

### PR DESCRIPTION
**Pull request by MaxGekk**
## What changes were proposed in this pull request?

In the PR, I propose to move `testFile()` to the common trait `SQLTestUtilsBase` and wrap test files in `AvroSuite` by the method `testFile()` which returns full paths to test files in the resource folder.

### CODE REVIEW

**Review by HyukjinKwon — APPROVED** [review]
LGTM

**Comment by MaxGekk** [comment]
jenkins, retest this, please

File: external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala [hunk 1]
```diff
@@ -37,27 +37,27 @@ import org.apache.spark.sql.test.{SharedSQLContext, SQLTestUtils}
 import org.apache.spark.sql.types._
 
 class AvroSuite extends QueryTest with SharedSQLContext with SQLTestUtils {
-  val episodesFile = "src/test/resources/episodes.avro"
-  val testFile = "src/test/resources/test.avro"
+  val episodesAvro = "episodes.avro"
+  val testAvro = "test.avro"
```
Comment (gengliangwang) [inline external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala]: Nit: 
Maybe we can make it simpler
```
val episodesAvro = testFile("episodes.avro")
val testAvro = testFile("test.avro")
```

**Comment by HyukjinKwon** [comment]
retest this please

**Review by gatorsmile — COMMENTED** [review]
LGTM 

Thanks! Merged to master

**Pull request closed without merging**